### ISSUE TITLE
add full support for aar files

### ISSUE THREAD

**Issue by AKBIROCK3** [mit-cml/appinventor-sources#2196]
**Describe the desired feature**

hi i wish you can add full support for aar files
https://developer.android.com/studio/build/manifest-merge

**Give an example of how this feature would be used**
like in android studio all the tags in androidmanifest.xml in the aar will be transfered to the main project that will help reduce the time and the work while creating compnents by keep us away from add all thses tag manually <!--

Explain why the use case cannot be completed using the features of the current system.:
because the current system doesn't support it

**Comment by ewpatton** [comment]
@pavi2410 @barreeeiroo One of you may want to think about supporting this as part of your GSOC projects.

**Comment by pavi2410** [comment]
I can take this up. But, does that mean that this should be added to the same design doc for the project I am currently doing?

**Comment by ewpatton** [comment]
@pavi2410 I would keep it separate so that it isn't on the critical path of the main part of your GSoC project. I would recommend you coordinate with @barreeeiroo because we have discussed refactoring the compiler logic to make updating it more manageable and this will have potential implications for that work.

**Comment by pavi2410** [comment]
I got it! 👍

**Comment by barreeeiroo** [comment]
Yes, we can surely talk and coordinate for something after finishing the main parts of our GSoC projects.
